package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "5.4.8",
+  "version": "6.0.0",
   "engines": {
-    "node": "^8.0.0 || ^10.13.0 || ^12.0.0"
+    "node": "^16.0.0"
```

Comment:
Should we make this `^16.14.2`?

Reply:
Related to the conversation below, I could see at least pinning to a minor version (middle number). So, `^16.14` or `^16.14.0`.